src/utils/changelog.ts

```diff
@@ -472,7 +472,7 @@ async function getPRAndMilestoneFromCommit(
         hash.slice(1),
         pr
           ? {
-              author: pr.author.login,
+              author: pr.author?.login ?? null,
```

Comment:
I don't think `?? null` is needed here with the `?.` access (since that'll become `undefined` already) -- I think this also needs a fix to the types above because as it is right now the `?.` looks unnecessary (I'm surprised the type checker isn't complaining about this?)